docs/volunteering/glossary.md

```diff
@@ -57,11 +57,6 @@ _Mentor_:
 
 : a volunteer who helps teams build their robots
 
-_PLOD_:
-
-: an initialism of [_Pink LED of Death_](/kit/hardware/servo-board.md#pink-led-of-death-plod-bug), a
-bug that is present in the Servo Board that can be identified by a pink / purple light on the board.
-
```

Comment:
Why is this being removed here? This content change doesn't seem related to the upgrade.

Reply:
The "Pink LED of Death" link doesn't exist, so it needs to go. Rather than removing the link, might as well remove the entire item, since the bug doesn't exist either.